hw/top_earlgrey/ip/ast/ast_wrapper.core

```diff
@@ -9,6 +9,8 @@ filesets:
     depend:
       - lowrisc:systems:ast_wrapper_pkg
       - lowrisc:systems:ast
+      - lowrisc:top_earlgrey:clkmgr
+      - lowrisc:systems:rstmgr_pkg
       - lowrisc:ip:entropy_src
       - lowrisc:prim:clock_buf
```

Comment:
* `lowrisc:prim:clock_buf` seems to be unused and can be removed.
* A `lowrisc:tlul:headers` dependency is missing for `tlul_pkg`.
* `lowrisc:ip:pwrmgr_pkg` should be added here.
* `lowrisc:prim:assert` is missing.

Reply:
Done. I've also added a dependency to `lowrisc:systems:sensor_ctrl_reg` which was missing.